repro.in

```diff
@@ -481,6 +481,7 @@ __END__
       --bind="$(readlink -e ${cachedir}):/var/cache/pacman/pkg" \
       --bind="$(readlink -e ${cachedir}):/cache" \
       bash -bash "${packages[@]}" <<-__END__
+set -e
```

Comment:
From what I understand this is an inline script, so it would be similar to having `set -e` at the start of a normal file.

Alternatively we can pass it as an argument to the shell itself, like so:

```sh
bash -ec 'false; echo hello'
# vs without:
#bash -c 'false; echo hello'
```
I'm not sure what `bash -bash ` does unfortunately.

Reply:
Ah sorry, I missed the section.